views/default/page/components/tabs.js

```diff
@@ -68,6 +68,7 @@ define(function (require) {
 				data: $link.data('ajaxQuery') || {},
 				beforeSend: function () {
 					changeTab($tab);
+					$target.html('');
```

Comment:
Are you sure about this?

Reply:
yes, after the ajax call is done, eighter it gets an error text of the result of the ajax call. SO no matter what the html content gets replaced by something else

Reply:
Maybe disable spinner and populate with ajax loader instead?

Reply:
i believe this is correct behaviour for page/components/tabs